specification/marketplacenotifications/resource-manager/Microsoft.MarketplaceNotifications/stable/2021-03-03/MarketplaceNotifications.json

```diff
@@ -0,0 +1,256 @@
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscription}/providers/Microsoft.MarketplaceNotifications/reviewsnotification": {
+      "get": {
+        "tags": [
+          "Notification"
+        ],
+        "operationId": "Notification_GetWithAuthorization",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/PrincipalId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/Notification"
+            }
+          },
+          "default": {
+            "description": "Microsoft.MarketplaceNotifications error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetReviewsNotification": {
+            "$ref": "./examples/GetReviewsNotification.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.MarketplaceNotifications/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Notification_GetOperations",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/AvailableOperations"
+            }
+          },
+          "default": {
+            "description": "Microsoft.MarketplaceNotifications error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetReviewsNotificationOperations": {
+            "$ref": "./examples/GetReviewsNotificationOperations.json"
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SubscriptionId": {
+      "description": "user's subscription id",
+      "type": "string",
+      "name": "subscription",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "PrincipalId": {
+      "description": "user's principal id",
+      "type": "string",
+      "name": "principalId",
+      "in": "query",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "ApiVersion": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The API version to use for the request.",
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "definitions": {
+    "Notification": {
```

Comment:
the resource must follow the standard resource schema with a top level "id", "name", and "type" and all resource specific properties in a "properties" object.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource

Reply:
@habenefr - This comment is not addressed. Can you update resource schema to follow ARM standard? The resource should look like below. Id, Name, Type are at top-level properties and rest of the fields should be under properties.

{
	"id": "/subscriptions/subId/providers/Microsoft.MarketplaceNotifications/reviewsNotification/notificationId",
	"type": "Microsoft.MarketplaceNotifications/reviewsNotification",
	"name": "notificationId",
	"properties": {
		"offerId": "publisher.offer",
		"date": "02-25-2021",
		"offerDisplayName": "Offer Name",
		"principalId": "0b12389c-5e1d-4b67-880c-71d0d4a8d751"	
	}
}